cSpell.json

```diff
@@ -402,9 +402,21 @@
             ]
         },
         {
-            "filename": "**/specification/migrate/resource-manager/Microsoft.Migrate/stable/2018-02-02/migrate.json",
+            "filename": "**/specification/migrate/resource-manager/**/*.json",
             "words": [
-                "Occured"
+                "Occured",
+                "Assessible",
+                "sqlcollectors",
+                "Reasonings",
+                "Overcommit",
+                "overcommit",
+                "SSDE",
+                "Recieved",
+                "Ebds",
+                "idms",
+                "Idms",
+                "Cluter",
```

Comment:
>Cluter [](http://example.com/codeflow?start=17&length=6)

Can we remove it?

Reply:
We have fixed any spelling mistake we could fix in our code. These ones are spelling mistakes in enums/ properties which have already been shipped. Now we cant change these properties as it would break the customers.